src/App.tsx

```diff
@@ -1,61 +1,133 @@
+import React, { useState } from 'react';
+import { TodoList } from './components/TodoList';
+import { Todo } from './Types/Todo';
 import './App.scss';
 
-// import usersFromServer from './api/users';
-// import todosFromServer from './api/todos';
+import { getUser } from './Helpers/getUser';
+
+import usersFromServer from './api/users';
+import todosFromServer from './api/todos';
+import { User } from './Types/User';
+
+export const App: React.FC = () => {
+  const [listOfTodos, setListOfTodos] = useState<Todo[]>(todosFromServer);
+  const [newTodoTitle, setnewTodoTitle] = useState<string>('');
+  const [selectedUser, setSelectedUser] = useState<number | string>(0);
```

Comment:
So a user with 0 id is initially selected?

Reply:
fixed 👌